### PR TITLE
Feature/auto update on login

### PR DESCRIPTION
**Pull request by Veladus**
Resolves IMAP-5

### CODE REVIEW

File: packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st [hunk 1]
```diff
@@ -6,9 +6,9 @@ connectAndLoginWith: anICAccountInfo
 	[self
 		connect;
 		login]
-	valueWithin: 2 seconds 
+	valueWithin: 5 seconds
```
Comment (shm0sby) [inline packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st:9]: What exactly does this change?

Comment (Veladus) [reply packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st:9]: OWA works

Comment (shm0sby) [reply packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st:9]: Please provide some more details...

Comment (Veladus) [reply packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st:9]: OWA is notoriously slow and for some users this provided problems, preventing login

Comment (shm0sby) [reply packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st:9]: So why did it work for Julia and me then?

**Pull request merged**